src/expand.rs

```diff
@@ -24,6 +26,20 @@ pub(crate) struct Project {
     workspace: PathBuf,
 }
 
+/// This `Drop` implementation will clean up the temporary crates when expansion is finished.
+/// This is to prevent pollution of the filesystem with dormant files.
```

Comment:
This seems to mean that every time `macrotest::expand` is called, all dependencies are re-compiled. For example, the following code will re-compile the dependencies three times, right?

```rust
#[test]
fn test() {
    macrotest::expand("tests/expand/a.rs");
    macrotest::expand("tests/expand/b.rs");
    macrotest::expand("tests/expand/c.rs");
}
```

If so, It may significantly increase the test time. (this is probably an unavoidable cost if the test is may conflicts)
Perhaps it seems preferable to adjust the documentation to recommend using wildcard(`*`) or expose a type like `trybuild::TestCases`.

Reply:
Makes sense! Yes, I'm afraid that dependencies will be recompiled for each set. I'll update the docs.